modules/serverless-testing-quarkus-functions.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-testing-quarkus-functions_{context}"]
 = Testing Quarkus functions
 
-You can test Quarkus functions locally on your computer by running the Maven tests that are included in the project template.
+You can test Quarkus functions locally on your computer. For that, run the Maven tests that are included in the project template.
```

Comment:
```suggestion
You can test Quarkus functions locally on your computer. Run the Maven tests that are included in the project template to test the functions.
```

Reply:
I'll leave this one as is. I think the transition in the beginning of the second sentence works better.